README.md

```diff
@@ -20,6 +20,7 @@ This Python package
 * Image references (uploaded as Confluence page attachments)
 * [Table of Contents](https://docs.gitlab.com/ee/user/markdown.html#table-of-contents)
 * [Admonitions](https://python-markdown.github.io/extensions/admonition/) (converted into *info*, *tip*, *note* and *warning* Confluence panels)
+* [Markdown in HTML](https://python-markdown.github.io/extensions/md_in_html/) 
```

Comment:
Unsure if we should add more docs around this or just a link to the plugin like you have done for admonition is good enough 🤷